tests/javascript/index.php

```diff
@@ -2549,6 +2549,10 @@ function callHistory(method)
             fireTemplateTag(templateToTest, params);
             var addedStyle4 = document.getElementById('customStyleTag4');
             strictEqual('var x = {};', addedStyle4.innerText, 'should have added element to start of head');
+            // Use attribute for compatibility.
+            var nonceAttr = addedStyle4.getAttribute('nonce');
+            nonceAttr = nonceAttr ? nonceAttr : addedStyle4.nonce;
```

Comment:
@snake14 was `addedStyle4.getAttribute('nonce')` failing in any case?

Reply:
@AltamashShaikh Yeah. In my local environment, it was failing until I used the property instead of the attribute.

Reply:
okay